src/ensembl/tests/fixtures/entity-viewer/transcript.ts

```diff
@@ -25,29 +27,30 @@ export const createTranscript = (): Transcript => {
     slice: transcriptSlice,
     exons: createExons(transcriptSlice),
     cds: createCDS(transcriptSlice),
-    product: createProduct()
+    product: createProduct(),
+    ...fragment,
   };
 };
 
 const createProduct = (): Product => {
   const length = faker.random.number({ min: 10, max: 100 });
-  const numberOfExons = faker.random.number({ min: 1, max: 10 });
-  const maxExonLength = Math.floor(length / numberOfExons);
+  const numberOfDomains = faker.random.number({ min: 1, max: 10 });
+  const maxDomainLength = Math.floor(length / numberOfDomains);
 
   const protein_domains_resources: ProteinDomainsResources = {};
 
-  times(numberOfExons, (index: number) => {
-    const minCoordinate = maxExonLength * index + 1;
-    const maxCoordinate = maxExonLength * index + 1;
+  times(numberOfDomains, (index: number) => {
+    const minCoordinate = maxDomainLength * index + 1;
+    const maxCoordinate = maxDomainLength * (index + 1);
     const middleCoordinate =
       maxCoordinate - (maxCoordinate - minCoordinate) / 2;
     const start = faker.random.number({
       min: minCoordinate,
-      max: middleCoordinate
+      max: middleCoordinate,
```

Comment:
We don’t. I have a suspicion that it may be Prettier that is adding them.

Reply:
Yes, as I suspected, Prettier has [changed](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker) its default behaviour since version 2.0: now the default is to add trailing commas. Many people prefer it, because it means smaller diffs in git commits). I added a config option to force the opposite to the current default.

Reply:
Thanks.
>Many people prefer it, because it means smaller diffs in git commits).
Sorry, what does this mean?

Reply:
Imagine you have an object:

```
{
  foo: 1,
  bar: 2
}
```

and in a subsequent commit you add a new field to it:

```
{
  foo: 1,
  bar: 2,
  baz: 3
}
```

Git diff for this will be 

```
-  bar: 2
+  bar: 2,
+  baz: 3
```

i.e. it will include the line with the field to which you had to add the comma to continue growing the object.

However, if you had a trailing comma:

```
{
  foo: 1,
  bar: 2,
}
```

the diff would only be:

```
+  baz: 3
```

So sufficiently many developers preferred this second scenario that Prettier decided to make it default.

_(I personally don't like the trailing comma, but solely on aesthetic reasons, which aren't an especially strong argument 😄 Would be happier if we didn't have it, but won't be particularly upset if we did.)_

Reply:
Thanks for that. I also prefer not to have it.